src/Blazored.Video/BlazoredVideo.razor.cs

```diff
@@ -183,7 +186,7 @@ async Task Implement(VideoEvents eventName)
 		/// We force one and a use it as a proxy for all the Media Events.
 		/// </summary>
 		/// <param name="args">The event args - Value contains our JSON</param>
-		void OnChange(ChangeEventArgs args)
+		protected virtual void OnChange(ChangeEventArgs args)
```

Comment:
Why was this changed?

Reply:
same as with `UniqueKey` but this time it was only my inner pattern sense that tickelt, as EventHandlers should been implemented protected virtual wherever possible to allow easy extension for somebody else.